mapreduce/tests/conftest.py

```diff
@@ -30,6 +61,30 @@ def mocked_auth_request():
         yield
 
 
+def setup_mapreduce():
+    # Run a job in order to get metrics from the environment
+    subprocess.Popen(
```

Comment:
```suggestion
    run_command(
```

Reply:
I'm not sure I can use `run_command`, since I want it to run in background. It just starts a job that will run when checking mapreduce, in order to have some metrics. I don't know if we have something for this use case